include/extnsionst.h

```diff
@@ -55,10 +55,10 @@ SOFTWARE.
 #include "privates.h"
 
 typedef struct _ExtensionEntry {
-    int index;
     void (*CloseDown) (         /* called at server shutdown */
                           struct _ExtensionEntry * /* extension */ );
     const char *name;           /* extension name */
+    int index;
```

Comment:
we first need to make sure the proprietary Nvidia driver isn't using it.

@dasha-uwu 

anyways, we don't have many of those, so not sure whether it's worth it.